tests/unit/test_dnb.py

```diff
@@ -54,19 +54,22 @@ def record(scrape_pos_page_body):
     return request.callback(response)
 
 
+@pytest.mark.skip(reason='connecting to external source')
```

Comment:
Can you add to the message in the skips a reference to the issue to fix it? (create the issue if not there yet please)